data/mods/gen3/random-teams.ts

```diff
@@ -572,7 +572,7 @@ export class RandomGen3Teams extends RandomGen4Teams {
 			// To prevent this, we prevent more than one Wobbuffet in a single battle.
 			if (set.ability === 'Shadow Tag') this.hasWobbuffet = true;
 		}
-		if (pokemon.length < 6 && !isMonotype) throw new Error(`Could not build a random team for ${this.format} (seed=${seed})`);
+		if (pokemon.length < 6 && !isMonotype) throw new Error(`Could not build a random team for ${this.format} (seed=${seed}) (team:${pokemon.map((mon) => mon.name)})`);
```

Comment:
The reason it doesn't show the team is because you can just repro with `./pokemon-showdown generate-team`. I don't think this adds anything.

Reply:
If it tells you the team right away you don’t even need to reproduce it, and that should save time.